addons/portal/wizard/portal_wizard.py

```diff
@@ -180,11 +180,11 @@ def _send_email(self):
             lang = wizard_line.user_id.lang
             partner = wizard_line.user_id.partner_id
 
-            portal_url = partner.with_context(signup_force_type_in_url='', lang=lang)._get_signup_url_for_action()[partner.id]
+            access_url = partner.with_context(signup_force_type_in_url='', lang=lang)._get_signup_url_for_action()[partner.id]
             partner.signup_prepare()
 
             if template:
-                template.with_context(dbname=self._cr.dbname, portal_url=portal_url, lang=lang).send_mail(wizard_line.id, force_send=True)
+                template.with_context(dbname=self._cr.dbname, access_url=access_url, lang=lang).send_mail(wizard_line.id, force_send=True)
```

Comment:
I think you should not changes anything here as it is a bit independant from the portal mixin.

Moreover I think you broke the template :p .